netbox/project-static/src/util.ts

```diff
@@ -409,7 +409,7 @@ export function createElement<
  * @returns Degrees in Fahrenheit.
  */
 export function cToF(celsius: number): number {
-  return celsius * (9 / 5) + 32;
+  return Math.round((celsius * (9 / 5) + 32 + Number.EPSILON) * 10) / 10;
```

Comment:
This seems a bit much. Shouldn't calling `round()` be sufficient?

Reply:
`round()` will just round the number to the next integer, don't we want to save that one decimal place?

also using `Number.EPSILON` to be more presise, take a look at this example:
```javascript
Math.round((1.005 + Number.EPSILON) * 100) / 100 // == 1.01
Math.round((1.005) * 100) / 100 // == 1
```